docs/src/gui/touchy.adoc

```diff
@@ -50,16 +50,16 @@ motion controller to control wheel jogging:
 'N' represents the axis number 0-8.
 
 - In addition to being connected to 'touchy.wheel-counts', the wheel counts
-  should also be connected to 'axis.N.jog-counts' for
-  each axis N.  If you use HAL component 'ilowpass' to smooth wheel jogging, be
-  sure to smooth only 'axis.N.jog-counts' and not 'touchy.wheel-counts'.
+  should also be connected to 'axis._n_.jog-counts' for
```

Comment:
In the pin descriptions of the man pages, the pins are always (as far as I can see) formatted like this:

axis._N_.jog-counts

Maybe we should do it here as well for consistency.
(Another point for the Style Guide...)

Reply:
You are right. Have converted back to upper-case Ns, slanted.